src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1340,48 +1337,48 @@
   "com.minecolonies.gui.visiblestatus.archer_combat": "Fighting",
   "com.minecolonies.gui.visiblestatus.archer_uni": "Shooting",
   "com.minecolonies.gui.visiblestatus.baker": "Baking",
-  "com.minecolonies.gui.visiblestatus.chickenherder": "Find chicken",
+  "com.minecolonies.gui.visiblestatus.chickenherder": "Finding chickens",
   "com.minecolonies.gui.visiblestatus.composter": "Composting",
   "com.minecolonies.gui.visiblestatus.cook": "Cooking",
-  "com.minecolonies.gui.visiblestatus.cowboy": "Herding Cows",
+  "com.minecolonies.gui.visiblestatus.cowboy": "Herding cows",
   "com.minecolonies.gui.visiblestatus.crusher": "Crushing",
   "com.minecolonies.gui.visiblestatus.delivery": "Delivering",
   "com.minecolonies.gui.visiblestatus.farmer": "Farming",
   "com.minecolonies.gui.visiblestatus.florist": "Gardening",
   "com.minecolonies.gui.visiblestatus.knight_combat": "Fighting",
   "com.minecolonies.gui.visiblestatus.miner": "Mining",
-  "com.minecolonies.gui.visiblestatus.lumberjack_search": "Searching Trees",
+  "com.minecolonies.gui.visiblestatus.lumberjack_search": "Searching for trees",
   "com.minecolonies.gui.visiblestatus.sawmill": "Carpentering",
   "com.minecolonies.gui.visiblestatus.smelting": "Smelting",
   "com.minecolonies.gui.visiblestatus.teacher_student": "Teaching",
 
-  "entity.fisherman.messagewatertoofar": "My hut seems to be too far away from the water, and I can't find any water to fish in! It should be an area of at least 7 x 7, and on the same height as my hut.",
+  "entity.fisherman.messagewatertoofar": "My hut seems to be too far away from water, and I can't find any water to fish in! It should be an area of at least 7x7 blocks, and the same height as my hut.",
   "com.minecolonies.coremod.furnaceuser.nofuel": "Please tell me what kind of fuel I should use in my furnace!",
   "com.minecolonies.coremod.furnaceuser.noore": "Please tell me what kind of ore I should smelt in my furnace!",
-  "com.minecolonies.coremod.florist.noplantground": "I don't have any composted dirt to plant on! Please repair or upgrade my building!",
-  "com.minecolonies.coremod.florist.nocompost": "I need compost to plant my flowers!",
-  "com.minecolonies.coremod.florist.noflowers": "I can't work, all flowers are disabled!",
-  "com.minecolonies.coremod.request.toolow": "I can't mine the %s at %d %d %d. You need to upgrade the building, so I can use a tool with higher mining level.",
+  "com.minecolonies.coremod.florist.noplantground": "%s: I don't have any composted dirt to plant on! Please repair my Flower Shop!",
```

Comment:
Since I believe these are displayed by the individual they apply to, do we need to add the %s here? It seems redundant.

Reply:
Oh, I thought this might've been displayed in chat. I'll fix this.